src/Algebra/Structures.agda

```diff
@@ -444,6 +451,12 @@ record IsSemiringWithoutOne (+ * : Op₂ A) (0# : A) : Set (a ⊔ ℓ) where
     ; ∙-congʳ to *-congʳ
     )
 
+  distribˡ : * DistributesOverˡ +
+  distribˡ = proj₁ distrib
+
+  distribʳ : * DistributesOverʳ +
+  distribʳ = proj₂ distrib
+
```

Comment:
These additions make sense, but if you are going to make them, then perhaps you should re-export them downstream from eg `IsRingWithoutOne` via opening the `isSemiringWithoutOne` manifest field... etc. which might simplify those exports elsewhere, too?